integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/KnnPredicateSpecificsIT.java

```diff
@@ -476,7 +472,7 @@ private static class IndexBinding {
 
 			public IndexBinding(IndexSchemaElement root, Collection<? extends VectorFieldTypeDescriptor<?>> fieldTypes,
 					VectorSimilarity vectorSimilarity) {
-				field = SimpleFieldModelsByType.mapAll( fieldTypes, root, "", c -> c.vectorSimilarity( vectorSimilarity ) );
+				field = SimpleFieldModelsByType.mapAll( fieldTypes, root, "", c -> c.projectable( Projectable.YES ).vectorSimilarity( vectorSimilarity ) );
```

Comment:
It would probably be best to use a different field to test projections. Otherwise you can't be sure knn works without `Projectable.YES`... We've had problems like that in the past, with `Sortable.YES` adding doc-values to the field, which turned out to be necessary for some predicates :x